src/global-firestore-data/user-semesters.ts

```diff
@@ -138,6 +184,48 @@ export const deletePlan = async (name: string, gtag?: VueGtag): Promise<void> =>
   store.commit('setCurrentPlan', store.state.plans[0]);
 };
 
+/** Add one course to multiple collections.
+ * This course is removed from the requirement choices.
+ */
+export const addCourseToCollections = (
+  plan: Plan,
+  year: number,
+  season: FirestoreSemesterSeason,
+  newCourse: FirestoreSemesterCourse,
+  collectionIDs: string[], // Array of collection IDs
+  gtag?: VueGtag
+): void => {
+  GTagEvent(gtag, 'add-course-collections');
+  editCollections(oldCollections =>
+    oldCollections.map(collection => {
+      if (collectionIDs.includes(collection.name)) {
+        return { ...collection, courses: [...collection.courses, newCourse] };
+      }
+      return collection;
+    })
+  );
+
+  deleteCourseFromSemester(plan, year, season, newCourse.uniqueID);
```

Comment:
I'd be curious about why we are necessarily removing something from a collection when we add from the dashboard main area, feels a bit counterintuitive. First time I tried it was annoyed that I lost the course and had to go remember what it was and find it.

Not an implementation question, more system design, would be interested to hear why this approach was taken.